recipes-bsp/u-boot/files/0011-arm-dts-iot2050-Disable-lock-step-mode-for-all-iot20.patch

```diff
@@ -0,0 +1,64 @@
+From b38646e913886c05ec35ccffe7071cfeed5b5161 Mon Sep 17 00:00:00 2001
+From: Li Hua Qian <huaqian.li@siemens.com>
+Date: Fri, 22 Mar 2024 10:24:26 +0800
+Subject: [PATCH] arm: dts: iot2050: Disable lock-step mode for all iot2050
+ boards
+
+The PG1 A variant of the iot2050 series has been identified which
+partially lacks support for lock-step mode. This implies that not all
+iot2050 boards can support this mode. As a result, lock-step mode has
+been disabled across all iot2050 boards for consistency and to avoid
+potential issues.
+
+Signed-off-by: Li Hua Qian <huaqian.li@siemens.com>
+---
+ arch/arm/dts/k3-am65-iot2050-common-pg2.dtsi | 5 -----
+ arch/arm/dts/k3-am65-iot2050-common.dtsi     | 5 +++++
+ arch/arm/dts/k3-am6528-iot2050-basic.dts     | 5 -----
```

Comment:
Same here: k3-am6548-iot2050-advanced-m2.dts

Reply:
It doesn't exist in k3-am6548-iot2050-advanced-m2.dts since no patch to do it in u-boot. I propose it was cleaned up during preparing https://github.com/siemens/meta-iot2050/commit/bc65912f96051c5bdf1b298a280a7efb84d14dfa.

Reply:
It does - in upstream, but that is due to a missing sync from the Linux kernel. Not relevant in this context.